spec/migrations/20150817213409_clear_tenant_seed_spec.rb

```diff
@@ -0,0 +1,26 @@
+require "spec_helper"
+require Rails.root.join("db/migrate/20150817213409_clear_tenant_seed")
+
+describe ClearTenantSeed do
+  class ClearTenantSeed::Tenant < ActiveRecord::Base
+  end
+  let(:tenant_stub) { migration_stub(:Tenant) }
+
+  migration_context :up do
+    it "works with no tenants" do
+      expect { migrate }.not_to raise_error
```

Comment:
Every migration spec has an implicit empty tables up and down test automatically, so no need for this spec.